libs/user_extensions/src/TTAlpsHistogramFiller.cpp

```diff
@@ -231,6 +231,93 @@ void TTAlpsHistogramFiller::FillJetHistograms(const shared_ptr<Event> event){
   }
 }
 
+void TTAlpsHistogramFiller::FillLooseDSAMuonsHistograms(const shared_ptr<Event> event){
+  float weight = GetEventWeight(event);
+  auto looseDSAMuons = event->GetCollection("LooseDSAMuons");
+
+  float nLooseDSAMuons = looseDSAMuons->size();
+  histogramsHandler->Fill("Event_nLooseDSAMuons", nLooseDSAMuons, weight);
+
+  for(auto dsaMuonObj : *looseDSAMuons){
+    float dxy = dsaMuonObj->Get("dxyPV");
+    float dz = dsaMuonObj->Get("dzPV");
+    float pt = dsaMuonObj->Get("pt");
+    float eta = dsaMuonObj->Get("eta");
+
+    histogramsHandler->Fill("LooseDSAMuons_dxy", dxy, weight);
+    histogramsHandler->Fill("LooseDSAMuons_dz", dz, weight);
+    histogramsHandler->Fill("LooseDSAMuons_pt", pt, weight);
+    histogramsHandler->Fill("LooseDSAMuons_eta", eta, weight);
```

Comment:
Could you check if we can just call "Get" directly in "Fill" call? If not, maybe like float(dsaMuonObj->Get("pt")?

Reply:
I see that you changed it, so I assume it works. Could you please do the same everywhere in this file?